requirements/core-requirements.txt

```diff
@@ -0,0 +1,11 @@
+alembic<2.*
```

Comment:
Can insert a comment that indicates this file is automatically generated and should not be modified manually?

Reply:
Great idea! Done :)